src/tools/qt-gui/qml/KeyWindow.qml

```diff
@@ -116,13 +116,13 @@ BasicWindow {
 					metaValueField.placeholderText: valuePlaceHolder
 
 					metaNameField.onTextChanged:  {
-						if(metaName !== metaNameField.text){
+                        if(metaName.trim() !== metaNameField.text.trim()){
```

Comment:
Is this trimming useful? Doesn't that mean metaNames cannot start and end with spaces?